inputWidget/1.html

```diff
@@ -5,8 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Input tag</title>
+    <link rel="stylesheet" href="1.css">
 </head>
 <body>
-    
+    <div class="parent">
+        <div class="num">Enter a number between 1 to 100</div>
```

Comment:
use 
<label> tag for writing the label for an input field. 

The <label> tag in HTML is used to provide a usability improvement for mouse users i.e, if a user clicks on the text within the <label> element, it toggles the control. The <label> tag defines the label for <button>, <input>, <meter>, <output>, <progress>, <select>, or <textarea> element